tests/README.md

```diff
@@ -0,0 +1,40 @@
+# Integration Test for Image-rs
+
+This integration test has 2 main sub-type tests:
+* Image decryption using [ocicrypt-rs](https://github.com/confidential-containers/ocicrypt-rs)
+* Image signature verification.
+
+## Image Decryption
+
+Implemented in `decrypt_layer.rs`.
+
+Image decryption will involve the image `"docker.io/arronwang/busybox_kbs_encrypted"`,
+following the steps:
+
+* Pull manifefst of `"docker.io/arronwang/busybox_kbs_encrypted"` without verification of signature.
```

Comment:
We can also link https://github.com/confidential-containers/image-rs/blob/main/test_data/generate_test_data.md here for how to generate KBS encrypted image